src/components/styles/PageWrapper.tsx

```diff
@@ -1,7 +1,7 @@
 import styled from "@emotion/styled/macro";
 
 export const PageWrapper = styled.div`
-  padding: 0 8px;
+  padding: 0 36px;
```

Comment:
we decided to add more padding for all the pages.

Reply:
bless